tests/integration/files/log_handlers/runtests_log_handler.py

```diff
@@ -35,8 +35,8 @@
 def __virtual__():
     if 'runtests_log_port' not in __opts__:
         return False, "'runtests_log_port' not in options"
-    if six.PY3:
-        return False, "runtests external logging handler is temporarily disabled for Python 3 tests"
+#    if six.PY3:
+#        return False, "runtests external logging handler is temporarily disabled for Python 3 tests"
```

Comment:
This is causing a lint error as the six library is no longer used

Reply:
This is only for testing.. It will not be commented out when WIP is removed from this pr